app/main.py

```diff
@@ -1,13 +1,19 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {
+        "line": (error["line_number"]),
```

Comment:
Why you need brackets here?
```suggestion
        "line": error["line_number"],
```

Reply:
The same in all places